auto_comps.py

```diff
@@ -0,0 +1,224 @@
+        for each in dragon_info_names_and_boardsize:
+            if each in lol_chess_tftnames_and_price[1]:
+                final_name = dragon_info_names_and_boardsize[each][0]
+                final_price = lol_chess_tftnames_and_price[1][each][0]
+                final_code_name = lol_chess_tftnames_and_price[1][each][1]
+                if each in HARD_OVERRIDE_LIST:
+                    final_code_name = HARD_OVERRIDE_LIST[each]
+                final_boardsize = dragon_info_names_and_boardsize[each][1]
+                comp_manager.champions[final_name] = {'Gold': final_price, 'Board Size': final_boardsize}
+        lol_chess_comps = __LoadLolChessComps(response_meta_comps, set_current, comp_manager)
+        comp_manager.comps_loaded = lol_chess_comps
+        jsoned_champions = json.dumps(comp_manager.champions)
+        jsoned_comps = json.dumps(comp_manager.comps_loaded)
+        with open(cached_file_path + set_current + '.json', 'w') as f:
+            f.write(jsoned_champions + '\n')
+            f.write(jsoned_comps)
+    print('Set: ' + set_current + ', loaded champions: ' + str(len(comp_manager.champions)) + ', comps: ' + str(
+        len(comp_manager.comps_loaded)))
+    for i in range(0, len(comp_manager.comps_loaded)):
+        temp = ','.join(comp_manager.comps_loaded[i][1])
+        print(str(i) + ' - ' + comp_manager.comps_loaded[i][0] + ' [' + temp + ']')
+
+    inputed = ''
+    temp_inputed = ''
+    inputed_file_path = os.path.join(cached_path, 'inputed')
+    if os.path.isfile(inputed_file_path):
+        with open(inputed_file_path, 'r') as f:
+            temp_inputed = f.read()
+        if temp_inputed != '':
+            print(
+                'Your last selection was: "' + temp_inputed + '", press Enter to use last selection, or type "n" and press Enter to make new selection')
+            if input().lower() == 'n':
+                inputed = ''
+            else:
+                inputed = temp_inputed
+
+    if inputed == '':
+        print(
+            'Select mode: \n-Press Enter to play random comps without sequence\n-Type "all" and press Enter to play all comps in sequence\n-Type "all_except 2 3 4"(for example) and press Enter to play all comps in sequence except selected\n-Type "1 2 3"(for example) and press Enter to play only selected comps in sequence (it will loop)')
+        inputed = input()
+    if inputed == '':
+        comp_manager.is_sequence_mode = False
+    elif inputed == 'all':
+        comp_manager.is_sequence_mode = True
+        for i in range(0, len(comp_manager.comps_loaded)):
+            comp_manager.sequence.append(i)
+    elif inputed.startswith('all_except'):
+        comp_manager.is_sequence_mode = True
+        splitted = inputed.split(' ')
+        temp_except = []
+        for each in splitted:
+            if each.isnumeric():
+                inted = int(each)
+                temp_except.append(inted)
+        for i in range(0, len(comp_manager.comps_loaded)):
```

Comment:
list comprehension on `temp_except`
```suggestion
        temp_except = [int(each) for each in splitted if each.isnumeric()]
        for i in range(len(comp_manager.comps_loaded)):
```